### ISSUE TITLE
Heroku migration

### ISSUE THREAD

**Issue by corbindavenport** [corbindavenport/imageshare#11]
Heroku is shutting down free dynos starting November 28, 2022, which is what ImageShare runs on: https://help.heroku.com/RSBRUH58/removal-of-heroku-free-product-plans-faq

I can't really justify $7/mo for ImageShare by itself, given I only use it infrequently myself and there aren't any donations coming in. It either needs to move to another host, or be rewritten as client-side JavaScript. I also can't redirect the current URL once I switch hosts after Nov 28, which is a problem. [Vercel](https://vercel.com/) may be an alternative.

I'll post any updates on the situation here, and on the [Discord server](https://discord.gg/59wfy5cNHw).

**Comment by corbindavenport** [comment]
I've set up ImageShare on DigitalOcean's App Platform, which is slightly cheaper ($5/mo) and didn't require any code changes. I've also set up a custom domain, [theimageshare.com](http://theimageshare.com/) so future hosting switches won't result in lost users.

Once everything is working, I'll enable an alert for people using the Heroku domain telling them to update their bookmarks. That should remain working until the Heroku shutdown date.
